### PR TITLE
split vdbench workload in perf ci

### PR DESCRIPTION
**Pull request by ebattat**
The main problem is that Perf CI is running more than 24 hours and that cause mixing data in Grafana dashboard 
This Pr split vdbench workload between odd and even week days because each vdbench run took ~1.5 hours.

1. Odd week days: **Sunday, Tuesday, Thursday, Saturday** - running vdbench scale workloads: 'vdbench_pod_scale', 'vdbench_kata_scale', 'vdbench_vm_scale'
2. Even week days: **Monday, Wednesday, Friday** - running vdbench workloads: 'vdbench_pod', 'vdbench_kata', 'vdbench_vm'

### CODE REVIEW

File: .github/workflows/Nightly_Perf_Env_Even_CI.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ name: Deploy Perf Env Nightly CI
 on:
   # Run on Nightly
   schedule:
-    - cron: '0 4 * * 0,2-6' # run Sunday, Tuesday-Sunday at 4 AM UTC/ 0 AM EDT
+    - cron: '0 4 * * 3,5' # run Sunday, Tuesday-Sunday at 4 AM UTC/ 0 AM EDT
```
Comment (athiruma) [inline .github/workflows/Nightly_Perf_Env_Even_CI.yml:9]: 0 4 * * 1,3,5
run Monday, Wednesday, and Friday at 4 AM UTC

File: .github/workflows/Nightly_Perf_Env_Odd_CI.yml [hunk 2]
```diff
@@ -0,0 +1,261 @@
+# Nightly CI https://github.com/marketplace/actions/deploy-nightly
+# This is a nightly CI Pipeline against Performance environment
+# GitHub Actions is limited 6 hours for each job
+name: Deploy Perf Env Nightly CI
+
+on:
+  # Run on Nightly
+  schedule:
+    - cron: '0 4 * * 0,2,4,6' # run Sunday, Tuesday-Sunday at 4 AM UTC/ 0 AM EDT
```
Comment (athiruma) [inline .github/workflows/Nightly_Perf_Env_Odd_CI.yml:9]: run  Sunday, Tuesday, Thursday, Saturday at 4 AM UTC/ 0 AM EDT ( even days )

File: .github/workflows/Nightly_Perf_Env_Even_CI.yml (hunk 1 above)
Comment (RobertKrawitz) [reply .github/workflows/Nightly_Perf_Env_Even_CI.yml:9]: Fix the comment, too.

File: .github/workflows/Nightly_Perf_Env_Odd_CI.yml [hunk 3]
```diff
@@ -0,0 +1,261 @@
+# Nightly CI https://github.com/marketplace/actions/deploy-nightly
```
Comment (RobertKrawitz) [inline .github/workflows/Nightly_Perf_Env_Odd_CI.yml:1]: BTW, could you arrange for the undesired workloads to be skipped depending upon the day of the week?  Something like

```
case "$(date +%w)" in
   0|3|5): ... ;;
   2|4|6): ... ;;
   *) skip all of the vdbench runs ;;
esac
```
so you don't need the second file that has to be kept in sync?

**Review by RobertKrawitz — CHANGES_REQUESTED** [review]

**Comment by ebattat** [comment]
We decided to run the CI 3 times a week: Monday, Wednesday, Friday.
Update it in new PR

**Comment by ebattat** [comment]
We decided to run the CI 3 times a week: Monday, Wednesday, Friday.
Update it in new PR

**Pull request closed without merging**